modules/site-health/admin-site-health.php

```diff
@@ -108,6 +108,94 @@ function ( &$value, $key ) {
+	 *
+	 * @param string                $key option name.
+	 * @param string|array|bool|int $value option value.
+	 * @return string|array $value option value.
+	 */
+	public function format_value( $key, $value ) {
+		$db_value = $value;
+		switch ( $key ) {
+			case 'browser':
+				if ( ! $value ) {
+					// translators: the placeholder is the option stored in database.
+					$value = esc_html__( 'Browser prefered language deactivated (0)', 'polylang' );
+					break;
+				}
+				// translators: the placeholder is the option stored in database.
+				$value = esc_html__( 'Browser prefered language activated (%s)', 'polylang' );
+				break;
+			case 'rewrite':
+				if ( 1 === $value ) {
+					// translators: the placeholder is the option stored in database.
+					$value = esc_html__( 'Remove /language/ in pretty permalinks (%s)', 'polylang' );
+				} else {
+					// translators: the placeholder is the option stored in database.
+					$value = esc_html__( 'Keep /language/ in pretty permalinks (%s)', 'polylang' );
+				}
+				break;
+			case 'hide_default':
+				if ( 1 === $value ) {
+					// translators: the placeholder is the option stored in database.
+					$value = esc_html__( 'Hide URL language information for default language (%s)', 'polylang' );
+				} else {
+					// translators: the placeholder is the option stored in database.
+					$value = esc_html__( 'Display URL language information for default language (%s)', 'polylang' );
+				}
+				break;
+			case 'force_lang':
+				switch ( $value ) {
+					case '0':
+						// translators: the placeholder is the option stored in database.
+						$value = esc_html__( 'The language is set from content (%s)', 'polylang' );
+						break;
+					case '1':
+						// translators: the placeholder is the option stored in database.
+						$value = esc_html__( 'The language is set from the directory name in pretty permalinks (%s)', 'polylang' );
+						break;
+					case '2':
+						// translators: the placeholder is the option stored in database.
+						$value = esc_html__( 'The language is set from the subdomain name in pretty permalinks (%s)', 'polylang' );
+						break;
+					case '3':
+						// translators: the placeholder is the option stored in database.
+						$value = esc_html__( 'The language is set from different domains (%s)', 'polylang' );
+						break;
+				}
+				break;
+			case 'redirect_lang':
+				if ( 1 === $value ) {
+					// translators: the placeholder is the option stored in database.
+					$value = esc_html__( 'The front page URL contains the language code instead of the page name or page id (%s)', 'polylang' );
+				} else {
+					// translators: the placeholder is the option stored in database.
+					$value = esc_html__( 'The front page URL contains the page name or page id instead of the language code (%s)', 'polylang' );
+				}
+				break;
+			case 'media_support':
+				if ( ! $value ) {
+					$value = esc_html__( 'The media are not translated (0)', 'polylang' );
+				}
```

Comment:
```suggestion
				}
				break;
```
Looks like you forget the `break`

Reply:
It's in line 185

Reply:
Oh yes! Thank you @Marianne380